MarkdownAsset.js

```diff
@@ -7,11 +7,16 @@ class MarkdownAsset extends Asset {
   constructor (name, pkg, options) {
     super(name, pkg, options)
     this.type = 'js'
-    this.md = new MarkdownIt('default', {
+    var md = new MarkdownIt('default', {
```

Comment:
I'd try to keep the scope in this.md, something like, I don't see the need increment the scope to `var md`: 
```
this.md = new MarkdownIt(...)
try {
      this.md = md.use(require('markdown-it-highlight').default)
} catch (error) {
     console.info('markdown-it-highlight not installed, ignoring...')
}
```
Flee free to try a generic solution for plugin loading, so we can use it also with `Meta`